rasa_nlu/classifiers/embedding_intent_classifier.py

```diff
@@ -396,6 +390,7 @@ def _create_batch_b(self, batch_pos_b, intent_ids):
         return np.concatenate([batch_pos_b, batch_neg_b], 1)
 
     def _linearly_increasing_batch_size(self, ep):
```

Comment:
let's use the function from embedding policy, that checks whether `self.batch_size` is a `list`

Reply:
Good catch, didn't realize this function was making unnecessary computations